lib/ain-evm/src/evm.rs

```diff
@@ -418,6 +436,11 @@ impl EVMServices {
                 .generate_logs_from_receipts(&receipts, block.header.number)?;
             self.receipt.put_receipts(receipts)?;
             self.filters.add_block_to_filters(block.header.hash());
+
+            self.channel
+                .sender
+                .send(Notification::Block(block.header.hash()))
+                .map_err(|e| format_err!(e.to_string()))?;
```

Comment:
This is an incorrect place to notify that a new block is created. This is still in consensus, and there could be downstream failures that may potentially happen. We should expose the send notification as an FFI, and call it at the end of connectblock pipeline on the CPP side.